### ISSUE TITLE
Gradle build fails with 3.7.0

### ISSUE THREAD

**Issue by joelfernandes19** [OneSignal/OneSignal-Android-SDK#395]
Version update `3.7.0` fails to build and gives the following error:
```
> Could not find com.google.android.gms:play-services-gcm:11.6.99.
Required by:
project :app > com.onesignal:OneSignal:3.7.0
> Could not find com.google.android.gms:play-services-location:11.6.99.
Required by:
project :app > com.onesignal:OneSignal:3.7.0
```

It is overriding Play Services version. I have the latest OneSignal Gradle plugin - `0.8.0` with the following dependency - `compile 'com.onesignal:OneSignal:[3.6.5, 3.99.99]'` Also, my Play Services version is set to the latest `11.6.0`

**Comment by amautone** [comment]
Same here, and I can't sync my gradle! Any ideas on how to force OneSignal 3.6.5? (at least to be able to compile my project)

**Comment by joelfernandes19** [comment]
@amautone I complied with the older version with `compile 'com.onesignal:OneSignal:3.6.5'`, and it synced fine. But fails with the latest version.

**Comment by amautone** [comment]
@joelfernandes19 thank you! I was using `'compile com.onesignal:OneSignal:[3.6.5, 3.99.99]'` but in this way was still compiling with the latest one (3.7.0) giving me error

**Comment by priyacse0054** [comment]
I was also having the same issue since morning, whereas my project was running fine till yesterday . Thanks @joelfernandes19  its working now.

**Comment by jkasten2** [comment]
Sorry for the issue, we just released a 3.7.1 version to fix the dependency ranges.

**Comment by joelfernandes19** [comment]
Works fine now. Thanks for the update @jkasten2
